winui/Rating/Getting-Started.md

```diff
@@ -1,12 +1,89 @@
 ---
 layout: post
-title: Getting Started with WinUI Rating control | Syncfusion
-description: Learn all about getting started with the Syncfusion WinUI Rating(SfRating) control, its elements, and more here.
+title: Getting started with WinUI Rating control | Syncfusion
+description: Learn here all about getting started with the Syncfusion WinUI Rating(SfRating) control and its basic features.
```

Comment:
description: Learn all about getting started with the Syncfusion WinUI Rating(SfRating) control and its basic features here.

Reply:
Changed the content description as mentioned above.